### DISCUSSION TITLE
Social Media Sharing

### DISCUSSION THREAD

**Discussion by thatvirtualboy** [mmistakes/jekyll-theme-basically-basic#120] [Q&A] [answered]
I feel like I'm missing something. Every time I share a post either to Twitter, LinkedIn, or any similar site, nothing other than the URL link gets posted.  For example, I can't get a Twitter Card with any additional content to populate, even with the author section added in _config.yml. 

Post images also don't appear to show up in any sharing. Am I doing something wrong? Or are these limitations of the theme? Thanks!

**Comment by mmistakes** [comment, marked as answer]
All social media related meta is handled by the [Jekyll SEO tag plugin](https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/usage.md). It's only as good as the post content you give it. 

You can use Twitter's card validator to see what sort of meta content it's picking up for a given post.

https://cards-dev.twitter.com/validator

I can confirm that the theme does work fine with cards when properly configured.

## Post without an `image`

![image](https://user-images.githubusercontent.com/1376749/76649460-18afc300-6537-11ea-8afb-ba85aff99cbd.png)

## Post with image

![image](https://user-images.githubusercontent.com/1376749/76649509-32510a80-6537-11ea-8341-2b9fe5ce8422.png)

**Comment by thatvirtualboy** [comment]
Thanks for the explanation. Apparently I had removed the `jekyll-seo-tag` from my config and didn't realize it 🥴
Also didn't realize the photo had to be a hero image. Appreciate the tip!
